src/api.ts

```diff
@@ -279,8 +295,12 @@ export const getStarredEvents = (userId: number) =>
 // Anyone can modify themselves
 // Only admins can modify other users
 export const modifyUser = (userId: number, user: EditableUser) =>
-  putRequest<null>(`/users/${userId}`, user)
+  patchRequest<null>(`/users/${userId}`, user)
```

Comment:
Is this patching with the full user? Because that basically defeats the purpose of the patch, and makes it a put.

Reply:
No, it doesn't have to be a full user. I forgot to switch this, cause there was a bug in the docs generator. I will add that.